src/Renderer.php

```diff
@@ -250,18 +250,18 @@ protected function createJobs()
      */
     protected function prepareRequest($jobs)
     {
-        $preparedJobs = array_map(function ($job) {
-            foreach ($this->plugins as $plugin) {
-                $job = $plugin->prepareRequest($job);
-            }
-            return $job;
-        }, $jobs);
+        $parsedJobs = $jobs;
```

Comment:
can I ask why you called this `parsedJobs`?  there's no parsing going on here

Reply:
Because I misread `prepared` will fix!